httlemon/collections.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from tinydb import TinyDB
+from tinydb.storages import JSONStorage
+
+
+DB_NAME = 'httlemon/httlemon_db.json'
+TABLE_COLLECTIONS_NAME = 'collections'
+DB_STORAGE = JSONStorage
+
+
+class InvalidCollectionName(Exception):
+    pass
+
+def add(collection_name):
+
+    if collection_name and isinstance(collection_name, str):
```

Comment:
can't we just _make_ it an instance of `str`? I'm not sure why to be so strict in the case I want a collection to just be called `1`